assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -82,7 +74,7 @@ const renderEdit = (attributes, toAttribute, setAttributes) => {
               label={__('Select pages', 'planet4-blocks-backend')}
               selected={posts || []}
               onChange={toAttribute('posts')}
-              postType={cover_type === COVERS_TYPES.content ? 'post,page' : 'act_page'}
+              postType={'act_page'}
```

Comment:
I think that this isn't working as expected, it seems all pages show up 🤔 But it's not due to your changes so I'm fine with sorting this out later

Reply:
Nice catch! Let's open a new ticket for this.

Reply:
I'm wondering if it shouldn't be `p4_action` instead of `act_page`

Reply:
Ah nevermind, `act_page` is what we use in the PostSelector component so that's fine

Reply:
Don't we need to check if `new_ia` is enabled here and then pass `p4_action` or `act_page` or am I wrong?

Reply:
Just curiously.. why do we remove `post` and `page`? Is that part of the requirement?

Reply:
We're removing `post,page` because this was only for Content Covers and we are retiring this style. I was also confused by this `act_page` but it's handled in [PostSelector](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/block-editor/PostSelector/index.js#L57-L67)

Reply:
Is the PostSelector behaves as expected, then? I'm sure we might include `p4_action` from [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/block-editor/PostSelector/index.js#L57).

Reply:
I'm not sure it's really working as expected because it seems that **all** pages show there, which I don't think is what we want:

<img width="268" alt="Screenshot 2024-12-03 at 14 02 00" src="https://github.com/user-attachments/assets/62a88f73-a4b5-4c1c-9f1a-65baa8e800bd">